common:
  rootDir: "/home/fyx/fastlio2/src/onine_relo"
  pointCloudTopic:  "/os1_points" #"/os_cloud_node/points" #"velodyne_points"    "points_raw"     rslidar_points      # Point cloud data
  imuTopic: "/imu/data" #"/os_cloud_node/imu" #"imu_data" #"imu_raw"          # IMU data
  use_lio: true

preprocess:
  lidar_type: 2                            # lidar sensor type, either 'Velodyne' or 'Ouster' 'Robosense' 'Livox'
  livox_type: 1                # 1 for LIVOX_CUS
  scan_line:  64
  blind: 4
  feature_enabled: false  # bool
  point_filter_num: 1 
  scan_rate: 10 

mapping:
  extrinsic_T: [1.77, -0.00, -0.05] # lidar to imu
  extrinsic_R: [ -1, 0, 0, 
                 0, -1, 0, 
                 0, 0, 1]
  acc_cov: 0.009939570888238808e-03
  gyr_cov: 0.005636343949698187e-03
  b_acc_cov: 0.64356659353532566e-03
  b_gyr_cov: 0.35640318696367613e-03
